interface/forms/newpatient/common.php

```diff
@@ -528,18 +533,18 @@ function cancelClickedOld() {
             <?php //can change position of buttons by creating a class 'position-override' and adding rule text-align:center or right as the case may be in individual stylesheets ?>
             <div class="form-row">
                 <div class="col-sm-12 text-left position-override">
-                    <button type="button" class="btn btn-secondary btn-save" onclick="top.restoreSession(); saveClicked(undefined);"><?php echo xlt('Save');?></button>
+                    <button type="button" class="btn btn-primary" onclick="top.restoreSession(); saveClicked(undefined);"><i class="fa fa-floppy-o mr-1" aria-hidden="true"></i><?php echo xlt('Save');?></button>
                     <?php if ($viewmode || empty($_GET["autoloaded"])) { // not creating new encounter ?>
-                        <button type="button" class="btn btn-link btn-cancel btn-separate-left" onClick="return cancelClickedOld()"><?php echo xlt('Cancel');?></button>
+                        <button type="button" class="btn btn-danger btn-cancel btn-separate-left" onClick="return cancelClickedOld()"><?php echo xlt('Cancel');?></button>
                     <?php } else { // not $viewmode ?>
-                    <button class="btn btn-link btn-cancel btn-separate-left link_submit" onClick="return cancelClickedNew()">
+                    <button class="btn btn-danger btn-cancel btn-separate-left link_submit" onClick="return cancelClickedNew()">
```

Comment:
per recent styling guidelines, lets:
1. use btn-secondary
2. remove btn-cancel and add the cancel icon (from https://github.com/openemr/openemr/blob/master/interface/themes/core/FontAwesome.scss#L29) to the button

Reply:
I confused in item 2 add the cancel icon,
cancel icon existed in FontAwesome.scss,

Do you mean to add a floppy icon?

Reply:
sorry, I should of been more clear on this.

the goal is to not use the btn classes used in https://github.com/openemr/openemr/blob/master/interface/themes/core/FontAwesome.scss

In this case the btn-cancel is using fontawesome icon `\f00d`, which is fa-times

So, will look something like:
```
<button type="button" class="btn btn-secondary btn-separate-left" onClick="return cancelClickedOld()"><i class="fa fa-times mr-1" aria-hidden="true"></i><?php echo xlt('Cancel');?></button>
```

Reply:
hi @stu01509 ,
Also, lets incorporate this rec from the styling guide:
If you have a group of two or more buttons, be sure to use the btn-group class

And will need to fix the merge conflict

(then this should be good for the codebase :) )

Reply:
@bradymiller 
Now why in the world and since when is it taboo to have standard save,delete and cancel buttons defined. We're going to be chasing our tails if ever we decide to change an icon. Plus why insist a dev have to do that extra work in design. Button design look and feel is part of the theme and should be defined in one place to keep theme consistent through out. Let's not get to carried away here.

I agree to not allow custom default button overrides for sizing, spacing etc or even overriding the button itself like theme light does a lot. Btn-save, btn-cancel, btn upload and on are all over core code and doing this is not a coding compromise but just makes sense.

Reply:
hi @sjpadgett ,
Just following the styling guide as discussed per @tywrenn . This is definitely the time to figure these things out and what you state is very, very, very reasonable (ie. I am ok with whatever the community decides in this regard), so rec discussing this issue there so we can come to a good centralized conclusion on this asap:
https://github.com/openemr/openemr/issues/3160

Reply:
hi @sjpadgett and @stu01509 ,
I posted this issue on the styling issue. The more I think about, the more what @sjpadgett states makes total sense. We can essentially just use the buttons classes simply for icons:
https://github.com/openemr/openemr/blob/master/interface/themes/core/FontAwesome.scss#L19-L110

Reply:
I'm gonna post some rec changes based on above